opensearch-sigv4/spec/spec_helper.rb

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+# Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Is this copied from Elsatic? New code doesn't need this license grant.

Reply:
Nice catch! Let me remove that!